advanced-usage/index.md

```diff
@@ -196,6 +196,14 @@ Documentation can also be parsed if
 {% end %}
 {{ easy_image(src="available-services", width=600, class="pirate") }}
 
+{% pirate() %}
+### Bag Editor
+
+The Bag Editor is a helper service for advanced users, with it is possible to change the
+database used to handle frontend interface changes.
+{% end %}
+{{ easy_image(src="bag-editor", width=600, class="pirate") }}
```

Comment:
You've linked to an image here, but seemingly forgot to include the actual image in the PR?